lib/v2/fuliba/latest.js

```diff
@@ -0,0 +1,50 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+
+module.exports = async (ctx) => {
+    const domain = ctx.query.domain ?? 'fuliba.net';
+    const res = await got(`https://${domain}`, {
```

Comment:
GHSA-64wp-jh9p-5cg2

Reply:
Any action I need to take, or? :)

Reply:
This introduce an obvious vector for SSRF attacks. Either remove it or it won't be merged.

Reply:
Got it, thanks. Then I will refine/re-submit there. 

> This introduce an obvious vector for SSRF attacks. Either remove it or it won't be merged.